### PR TITLE
Added ARC/BRD skills and statuses; also fixed a typo on "CRITIAL UP"

### PR DESCRIPTION
**Pull request by jplippi**

### CODE REVIEW

File: src/registerServiceWorker.js [hunk 1]
```diff
@@ -8,112 +8,112 @@
...
-}
-
-export function unregister() {
-  if ('serviceWorker' in navigator) {
-    navigator.serviceWorker.ready.then(registration => {
-      registration.unregister();
-    });
-  }
-}
+/* eslint-disable */
```
Comment (ackwell) [inline src/registerServiceWorker.js]: This reeks of line ending issues :notlikeblob:

File: src/parser/jobs/brd/About.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import ACTIONS from 'data/ACTIONS'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
+		<p>Welcome to the Bard module! Despite being a very straightforward job, Bard's complexity is deceiving.</p>
+		<p>Considered by many as an <i>"easy to learn, hard to master"</i> job, Bard is a job that relies heavily on decision-making.</p>
+		<p>Improvements on Bard can range from the fundamentals of properly utilizing songs (<ActionLink {...ACTIONS.THE_WANDERERS_MINUET}/>, <ActionLink {...ACTIONS.MAGES_BALLAD}/> and <ActionLink {...ACTIONS.ARMYS_PAEON}/>) up to the intricacies of <ActionLink {...ACTIONS.IRON_JAWS}/> and the concept of buff/debuff snapshotting.</p>
+		<p>This analyzer will guide you through the job's core mechanics, all the way to encounter-specific optimization.</p>
+
+		<Message warning icon>
```
Comment (ackwell) [inline src/parser/jobs/brd/About.js]: Nice description, I really like it!

Can I just get a copy update for the warning please, I think it's still got a bit of a ways to go 😓

File: src/parser/jobs/brd/SongUptime.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	}
+
+	_onComplete() {
+
+		const fightDuration = (this.parser.fightDuration - this.downtime.getDowntime())/1000
+		const songlessTime = (this._getSonglessTime())/1000
+		const songlessPercentile = (songlessTime/fightDuration)*100
+
+		// Debug info
+		console.log('Fight duration: ' + fightDuration + ', Songless time: ' + songlessTime + ', Songless percentage: ' + songlessPercentile)
```
Comment (ackwell) [inline src/parser/jobs/brd/SongUptime.js]: Comment out or guard please

File: src/parser/jobs/brd/SongUptime.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+			const deathEvent = this._deathEvents.find( d => d.timestamp > this._songCastEvents[i].timestamp)
+
+			// If death was before the theoretical songless period
+			if (deathEvent ? deathEvent.timestamp < songless.start : false) {
+				// Then death marks the start of the songless period
+				songless.start = deathEvent.timestamp
+			}
+
+			// Debug info
+			console.log('Songless start: ' + this._formatTimestamp(songless.start) + ', Songless end: ' + this._formatTimestamp(songless.end))
```
Comment (ackwell) [inline src/parser/jobs/brd/SongUptime.js]: Same dealio

File: src/parser/jobs/brd/SongUptime.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+				const effectiveSonglessTime = Math.max(theoreticalSonglessTime - this.downtime.getDowntime(songless.start, songless.end), 0)
+				totalSonglessTime += effectiveSonglessTime
+			}
+		}
+
+		return totalSonglessTime
+	}
+
+	// Method for debug purposes. Converts a timestamp into a time in seconds relative to the start of the encounter
+	_formatTimestamp(timestamp) {
```
Comment (ackwell) [inline src/parser/jobs/brd/SongUptime.js]: `this.parser.formatTimestamp(...)`?

File: src/parser/jobs/brd/Songs.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
+//import React, {Fragment} from 'react'
```
Comment (ackwell) [inline src/parser/jobs/brd/Songs.js]: This file seems to be duping up on `songuptime`. Not really doing much, worth keeping around at the moment?

File: src/registerServiceWorker.js (hunk 1 above)
Comment (jplippi) [reply src/registerServiceWorker.js]: I think this came from Vay's branch, no idea what it actually is lol
I did have line ending issues, but fixed it without disabling lint

File: src/parser/jobs/brd/SongUptime.js (hunk 5 above)
Comment (jplippi) [reply src/parser/jobs/brd/SongUptime.js]: D-does that exist? :blobsweats:

File: src/parser/jobs/brd/Songs.js (hunk 6 above)
Comment (jplippi) [reply src/parser/jobs/brd/Songs.js]: That's also from Vay, I believe this module is still wip

File: src/parser/jobs/brd/SongUptime.js (hunk 3 above)
Comment (jplippi) [reply src/parser/jobs/brd/SongUptime.js]: Ooops!

File: src/registerServiceWorker.js (hunk 1 above)
Comment (Vayhyr) [reply src/registerServiceWorker.js]: The file is fine on my branch, someone changed that file today https://i.imgur.com/GgpMAbE.jpg

File: src/parser/jobs/brd/About.js (hunk 2 above)
Comment (jplippi) [reply src/parser/jobs/brd/About.js]: Uh, I think I changed this one? It's different in my working copy huh

**Pull request merged**